pcdsdevices/signal.py

```diff
@@ -116,30 +122,61 @@ class FakePytmcSignalRO(FakePytmcSignal, FakeEpicsSignalRO):
 fake_device_cache[PytmcSignal] = FakePytmcSignal
 
 
+def is_meek_ophyd_object(obj: ophyd.ophydobj.OphydObject):
+    """
+    Can you expect the ophyd object to ever give subscription callbacks?
+
+    If not, pcdsdevices considers that object to be "meek" or quiet.  This can
+    be useful to determine whether or not you should wait on a subscription
+    callback to happen before continuing.
+    """
+    if not isinstance(obj, ophyd.ophydobj.OphydObject):
+        raise ValueError(f"{obj} is not an ophyd object")
+
+    return any(isinstance(obj, cls) for cls in fake_device_cache.values())
```

Comment:
I don't quite understand this implementation. An ophyd device is quiet if and only if it is fake?

Reply:
I'm open to suggestions on this one, but the goal is to find a way of determining whether or not a given signal (or object) will _ever_ give a value subscription without being poked at.

In the above case, I used a shortcut of "if it's a FakeEpicsSignal or a FakePytmcSignal or ..." it has no control layer underneath it so it will not be getting any value callbacks without a specific `put()` in the right spot in the test suite.